vendor/k8s.io/client-go/util/certificate/OWNERS

```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
This module is under util/, it may change drastically with newer version of client-go. Maybe we should clone it under under repo instead.

Reply:
Let's not cloning, if it changes we adapt, we can always go back to clone old version if needed.